hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/impl/ITestConnectionTimeouts.java

```diff
@@ -156,7 +159,12 @@ public void testGeneratePoolTimeouts() throws Throwable {
     ContractTestUtils.createFile(fs, path, true, DATASET);
     final FileStatus st = fs.getFileStatus(path);
     try (FileSystem brittleFS = FileSystem.newInstance(fs.getUri(), conf)) {
-      intercept(ConnectTimeoutException.class, () -> {
+      Class exceptionClass = ConnectTimeoutException.class;
```

Comment:
won't be needed once the translation is fixed.

Reply:
ack